src/components/ranking/RankingSection.tsx

```diff
@@ -98,15 +145,23 @@ const RankingSection = () => {
         onChange={(value) => handleParamChange('rank', value)}
       />
 
-      <ProductGrid
-        products={rankingProducts}
-        showMore={showMore}
-        onProductClick={handleProductClick}
-      />
+      {isLoading ? (
+        renderLoadingSkeleton()
+      ) : error ? null : ( // 에러시 아무것도 표시하지 않음
```

Comment:
삼항연산이 이중으로 들어가있어 코드를 읽을때 혼동을 줄 수 있을 것 같아요.
if문이 많이 들어가더라도 읽기 쉬운 아래와 같은 방법을 추천드립니다!


```suggestion
if(isLoading){
   return renderLoadingSkeleton();
}

if(error){
  return null;
}
```

Reply:
수정하였습니다.
이 코드처럼 더 짧게 작성할 수 있는 코드를 가독성을 위해 어느정도로 펼쳐서 작성해야 할지 고민될 때가 많은데, 협업을 고려하면 그냥 다 펼쳐서 if문으로 쉽게 적는게 더 나을거같네요. 제안 감사합니다!